tests/ManageCourses.Tests/DbIntegration/DataServiceTests.cs

```diff
@@ -254,6 +254,26 @@ public void GetCoursesShouldReturnData()
                 Assert.AreEqual(numCourses, result.Courses.Count);
             }
         }
+        [Test]
+        public void GetCoursesShouldReturnStatus()
+        {
+            const int numOrgs = 5;
+            const int numCourses = 6;
+            LoadData(TestUserEmail1, numOrgs, numCourses);
+            var orgList = DataService.GetOrganisationsForUser(TestUserEmail1).ToList();
+            Assert.IsTrue(orgList.Count == numOrgs);
+            Assert.IsTrue(orgList.All(c => c.TotalCourses == numCourses));
```

Comment:
Use fluent assertions throughout for nicer syntax and better test failure messages.

Reject.

Reply:
All done